tools/sigclient/cmd/sigclient.go

```diff
@@ -141,6 +142,15 @@ var esQueryCmd = &cobra.Command{
 		log.Infof("filePath : %+v\n", filepath)
 		log.Infof("randomQueries: %+v\n", randomQueries)
 		log.Infof("bearerToken : %+v\n", bearerToken)
+
+		if cicd {
+			err := query.MigrateLookupsForCicd()
```

Comment:
Naming the flag `cicd` is a bit confusing; it's not immediately obvious what this would mean, and from that name I wouldn't guess what it actually does (move a lookup file).

Maybe a better way to do this would be to have a flag like `--lookups file1.csv file2.csv ...` (for now we could just support one file, if that's easier and we don't currently need to have multiple).

Also, you should add new command line options to the sigclient README